desktop-app/app/reducers/statusBar.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import {SET_STATUS_BAR_VISIBILITY} from '../actions/statusBar';
+import {statusBarSettings} from '../settings/statusBarSettings';
+
+export type StatusBarStateType = {visible: boolean};
+
+export default function app(
+  state: StatusBarStateType = {visible: statusBarSettings.getVisibility()},
```

Comment:
I don't feel entirely comfortable calling the settings to initialize the reducer state, but I can't see any other way to do this right now.

Reply:
@falecci I think it is fine. :)